portfolio.html

```diff
@@ -16,62 +16,82 @@
     </nav>
 
     <main>
-        <div class="ice">
-        </div>
+        <div class="fantasy">
+            <section class="container2">
+                <article class="mid-2">
+                    <h2>- Portfolio -</h2>
+                    <p>
```

Comment:
You don't need to move text onto a new line with a <p> tag, can just let it wrap the text